docs/user_guide/mitigation.rst

```diff
@@ -80,11 +81,408 @@ Reductions
+require only black-box access to a cost-sensitive classification algorithm,
+which does not need to have any knowledge of the desired definition of
+fairness or sensitive features. [#1]_ extends this to regression scenarios.
+
+From an API perspective this looks as follows in all situations
+
+>>> reduction = Reduction(estimator, constraints, **kwargs)  # doctest: +SKIP
+>>> reduction.fit(X_train, y_train, sensitive_features=sensitive_features)  # doctest: +SKIP
+>>> reduction.predict(X_test)  # doctest: +SKIP
+
+Fairlearn doesn't impose restrictions on the referenced :code:`estimator`
+other than the existence of :code:`fit` and :code:`predict` methods.
+At the moment, the :code:`estimator`'s :code:`fit` method also needs to
+provide a :code:`sample_weight` argument which the reductions techniques use
+to reweight samples.
+In the future Fairlearn will provide functionality to handle this even
+without a :code:`sample_weight` argument.
+
+Before looking more into the reductions approaches themselves this section
+examines the different constraints that are compatible with them. All of them
+are expressed as moments inheriting from :code:`ClassificationMoment` or
+:code:`LossMoment`, which in turn inherit from the base class :code:`Moment`.
+:code:`Moment`'s main purpose is to calculate the constraint violation of a
+current set of predictions through its :code:`gamma` function as well as to
+provide :code:`signed_weights` that are used to relabel and reweight samples.
+
+.. _constraints_binary_classification:
+
+Constraints for binary classification
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+All existing constraints for binary classification inherit from
+:code:`ClassificationMoment` and can be formulated as difference or ratio-based
+constraints.
+
+Difference-based constraints consider the difference with respect
+to a metric between a group of samples and the overall population or
+cohorts thereof.
+Parity represents the state where
+
+.. math::
+
+    \text{metric}_a - \text{metric}_* = 0 \; \forall a
+
+where :math:`a` is a group indentifier and :math:`*` represents the overall
+population. This is internally represented as two inequalities
+
+.. math::
+
+    \text{metric}_a - \text{metric}_* \leq 0 \; \forall a \\
+    -\text{metric}_a + \text{metric}_* \leq 0 \; \forall a
+
+The algorithms do not consider every kind of violation of these constraints,
+but rather violations that go beyond their :code:`difference_bound`. Such
+a difference-based constraint could be instantiated as
+
+    >>> ClassificationMoment(difference_bound=0.01)  # doctest: +SKIP
+
+Note that achieving parity with the :code:`difference_bound` does not mean
+that the difference between the group with the highest metric value and the
+group with the lowest metric value is below the :code:`difference_bound`, but
+rather the difference between each individual group and the metric value for
+the overall population (or cohort thereof) is below :code:`difference_bound`. 
```

Comment:
'or a cohort thereof' ? What's going on with that?

Reply:
Well, sometimes you don't evaluate against the overall population, but a cohort thereof. Example: Equalized Odds evaluates positives (one cohort) and negatives (another cohort) separately.

Reply:
OK, understood.